packages/wrangler/src/dev.tsx

```diff
@@ -264,6 +265,12 @@ export async function startDev(args: ArgumentsCamelCase<DevArgs>) {
 			((args.script &&
 				findWranglerToml(path.dirname(args.script))) as ConfigPath);
 		let config = readConfig(configPath, args);
+		metrics.sendMetricsEvent(
+			"run dev",
+			config,
```

Comment:
I'm uncomfortable sending a full config object here, can we be more intentional?

Reply:
Actually, I am going to rework this API a bit...